### PR TITLE
Add active state to selective recording icon in prime themes

### PR DESCRIPTION
**Pull request by gettinToasty**

### CODE REVIEW

File: app/services/layout/layout-data.ts [hunk 1]
```diff
@@ -37,49 +37,49 @@ type ILayoutData = {
 
 export const LAYOUT_DATA: ILayoutData = {
   [ELayout.Default]: {
-    resizeDefaults: { bar1: 156, bar2: 240 },
+    resizeDefaults: { bar1: 20, bar2: 40 },
```
Comment (holiber) [inline app/services/layout/layout-data.ts:40]: why the numbers are so different. Are they percentages now?

Comment (gettinToasty) [reply app/services/layout/layout-data.ts:40]: whoops those changes werent meant to be carried over, will fix

File: app/components/editor/elements/SourceSelector.vue [hunk 2]
```diff
@@ -135,7 +157,7 @@ i.disabled {
 }
 
 .icon--active {
-  color: var(--teal);
+  color: var(--icon-active);
```
Comment (andycreeth) [inline app/components/editor/elements/SourceSelector.vue:160]: Lol it's maybe slightly counterintuitive that `--teal` is set to `@goldenrod` in `themes.g.less`.  We should probably rename that variable at some point.

Comment (gettinToasty) [reply app/components/editor/elements/SourceSelector.vue:160]: yea i don't disagree-- maybe something like `--brand-primary` or `--primary-accent` would be better

**Pull request merged**